terminate called after throwing an instance of 'std::bad_alloc'
  what():  std::bad_alloc
Command terminated by signal 6
	Command being timed: "timeout 30h ./wcsd zhishihudong_degree V4 5 1"
	User time (seconds): 0.03
	System time (seconds): 0.19
	Percent of CPU this job got: 19%
	Elapsed (wall clock) time (h:mm:ss or m:ss): 0:01.17
	Average shared text size (kbytes): 0
	Average unshared data size (kbytes): 0
	Average stack size (kbytes): 0
	Average total size (kbytes): 0
	Maximum resident set size (kbytes): 114488
	Average resident set size (kbytes): 0
	Major (requiring I/O) page faults: 11
	Minor (reclaiming a frame) page faults: 28250
	Voluntary context switches: 6692
	Involuntary context switches: 53187
	Swaps: 0
	File system inputs: 107624
	File system outputs: 8
	Socket messages sent: 0
	Socket messages received: 0
	Signals delivered: 0
	Page size (bytes): 4096
	Exit status: 0
